modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -74,18 +75,26 @@ def fetch_cached_or_service(validated_params)
             start_date: [validated_params[:start_date], one_year_ago].min,
             end_date: [validated_params[:end_date], one_year_from_now].max
           )
+
           Mobile::V0::Appointment.set_cached(@current_user, appointments)
           Rails.logger.info('mobile appointments service fetch', user_uuid: @current_user.uuid)
         end
 
+        unless validated_params[:included]&.include?('pending')
```

Comment:
Fetching the requests is feature flagged in the proxy, but this should help the front end devs to test on deployed environments. Once the feature flag is turned on, we need to always fetch the appointment requests to maintain cache integrity.